docs/overrides/main.html

```diff
@@ -3,5 +3,90 @@
 {% block content %}
 {{ super() }}
 <script src="/flarelytics/client.js"></script>
+<script>
+    if (!window.__posthogInitialized) {
```

Comment:
it seems impossible for `window.__posthogInitialized` to be true here since this doesn't happen in a callback or anything

Reply:
moved into callback